platform/src/common/components/Collocation/Report/Charts/CorrelationChart.jsx

```diff
@@ -0,0 +1,112 @@
+import {
+  LineChart,
+  Line,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+  ResponsiveContainer,
+} from 'recharts';
+import moment from 'moment';
+
+const CorrelationChart = ({
+  pmConcentration,
+  hasCustomLegend,
+  CustomLegend,
+  data,
+  isInterSensorCorrelation,
+}) => {
+  let newData = [];
+
+  const colors = ['#8884d8', '#82ca9d', '#ffc658', '#ca82cc']; // Colors for each line
+
```

Comment:
Plotting sensor lines for only one device in intersensor correlation at the moment. Plotting sensor lines for more than one device is still challenging.